static/app/views/projectInstall/createProject.tsx

```diff
@@ -260,12 +261,29 @@ function CreateProject() {
   const canCreateTeam = organization.access.includes('project:admin');
   const isOrgMemberWithNoAccess = accessTeams.length === 0 && !canCreateTeam;
 
-  const canSubmitForm =
-    !inFlight &&
-    (team || isOrgMemberWithNoAccess) &&
-    canCreateProject &&
-    projectName !== '' &&
-    (!shouldCreateCustomRule || conditions?.every?.(condition => condition.value));
+  const isMissingTeam = !isOrgMemberWithNoAccess && !team;
+  const isMissingProjectName = projectName === '';
+  const isMissingAlertThreshold =
+    shouldCreateCustomRule && !conditions?.every?.(condition => condition.value);
+
+  const formErrorCount = [
+    isMissingTeam,
+    isMissingProjectName,
+    isMissingAlertThreshold,
+  ].filter(value => value).length;
```

Comment:
```suggestion
  const hasFormErrors = [
    isMissingTeam,
    isMissingProjectName,
    isMissingAlertThreshold,
  ].every(value => !value)
```

an alternative 🙂

Reply:
hmm, I still need the count so I can display `Please fill out all the required fields` if there is more then 1 error, right?

Reply:
I see...yes makes sense 👍